geode-core/src/test/java/org/apache/geode/internal/admin/remote/DistributionLocatorIdJUnitTest.java

```diff
@@ -46,4 +48,25 @@ public void testEquals() throws UnknownHostException {
 
   }
 
+  @Test
+  public void testEquals_and_DetailCompare() throws UnknownHostException {
+    DistributionLocatorId dLI1 = new DistributionLocatorId(40404, "127.0.0.1", null);
+    DistributionLocatorId dLI2 =
+        new DistributionLocatorId(40404, "127.0.0.1", "127.0.1.0", "member2");;
+    DistributionLocatorId dLI3 = new DistributionLocatorId(40404, "127.0.0.1", null, "member3");
+    DistributionLocatorId dLI4 = new DistributionLocatorId(dLI3.marshal());
+
+    assertTrue(dLI1.equals(dLI2));
```

Comment:
Can you please update the test to use AssertJ instead of JUnit Asserts? If these assertions ever fail, they will provide much better details for fixing the code if they're changed to:
```
assertThat(dLI1).isEqualTo(dLI2);
```